react-components/src/components/RevealToolbar/Help/MouseNavigation.tsx

```diff
@@ -29,18 +29,21 @@ export const MouseNavigation = ({ fallbackLanguage }: MouseNavigationProps): Rea
       )}>
       <MouseNavigationInstructionGrid>
         <InstructionText>{t('MOUSE_ZOOM', 'Zoom / scroll')}</InstructionText>
-        <InstructionText style={{ marginBottom: 30, textAlign: 'right' }}>
+        <InstructionText
+          style={{ marginBottom: 30, textAlign: 'right', width: 'max-content', maxWidth: '100px' }}>
           {t('MOUSE_ROTATE', 'Rotate')}
           <InstructionDetail>{t('MOUSE_INSTRUCTIONS', 'Click+drag')}</InstructionDetail>
         </InstructionText>
         <MouseNavigationCombinedGridItem>
           <StyledMouse />
         </MouseNavigationCombinedGridItem>
-        <InstructionText style={{ marginBottom: 30, textAlign: 'left' }}>
+        <InstructionText
+          style={{ marginBottom: 30, textAlign: 'left', width: 'max-content', maxWidth: '100px' }}>
           {t('PAN', 'Pan')}
           <InstructionDetail>{t('MOUSE_INSTRUCTIONS', 'Click+drag')}</InstructionDetail>
         </InstructionText>
-        <InstructionText style={{ marginTop: -50, textAlign: 'right' }}>
+        <InstructionText
+          style={{ marginTop: -50, textAlign: 'right', width: 'max-content', maxWidth: '100px' }}>
           {t('MOUSE_SELECT', 'Select Objects')}
```

Comment:
Should this be part of this PR?

Reply:
It make sense as Japanese, Korean & Chinese language string exposed the need to it